src/services/listService.js

```diff
@@ -9,8 +9,18 @@ export const deleteItem = (token, itemName) => {
   // delete field from firestore doc
   db.collection('lists')
     .doc(token)
-    .update({
-      [normalizedName]: firebase.firestore.FieldValue.delete(),
+    .collection('items')
+    .doc(normalizedName)
+    .delete()
+    .then(function () {
+      console.log('Item successfully deleted!');
+      alert(`"${itemName}" has been deleted from your list.`);
+    })
+    .catch(function (error) {
+      console.log('Error deleting document: ', error);
+      alert(
+        `Sorry, "${itemName}" cannot be deleted from your list at this time.`,
+      );
```

Comment:
Good work adding error handling for deleting items! The alerts might need a refactor for our new modals but we can handle that on merge day.